### PR TITLE
CLDR-14325 add a USERS.md file listing cldr-json users

### PR DESCRIPTION
**Pull request by srl295**
CLDR-14325

obviously there are others, but this is just to get it started

### CODE REVIEW

**Comment by srl295** [comment]
> @srl295 Thanks! Typo: `@phenseley` should be `@phensley`

woops! fixed?

**Comment by phensley** [comment]
Yep looks good

**Comment by longlho** [comment]
Can you add formatjs as well?

**Comment by srl295** [comment]
> Can you add formatjs & globalizejs as well?

How should it be worded? Can you suggest text and link?

File: USERS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cldr-json Users
+
+Here is a list of known users of cldr-json.
+
+Also see the main list of CLDR users on <http://cldr.unicode.org>
+
+## Projects
+
+- [cldr engine (@phensley/cldr)](https://phensley.github.io/cldr-engine)
+- [unicode.org/icu4x](https://github.com/unicode-org/icu4x)
```
Comment (longlho) [inline USERS.md:10]: - [formatjs](https://formatjs.io/)
- [globalizejs](https://github.com/globalizejs/globalize)

Comment (srl295) [reply USERS.md:12]: done

**Review by macchiati — APPROVED** [review]

**Comment by srl295** [comment]
OK that's it for now, please open more PRs as needed!

**Pull request merged**